napari/components/overlays/labels_polygon.py

```diff
@@ -0,0 +1,43 @@
+from napari.components.overlays.base import SceneOverlay
+from napari.layers import Labels
+
+
+class LabelsPolygonOverlay(SceneOverlay):
+    """Overlay that displays a polygon on a scene.
+
+        It handles the following mouse events to update the overlay:
+        - Mouse move: Continuously redraw the latest polygon point with the current mouse position.
+        - Mouse press (left button): Adds the current mouse position as a new polygon point.
+        - Mouse double click (left button): If there are at least three points in the polygon and
+                      the double-click position is within completion_radius from the first vertex,
+                      the polygon will be painted in the image using the current label.
+
+        - Mouse press (right button): Removes the most recent polygon point from the list.
```

Comment:
Updated to have 4 space indentation (Per PEP257) and wrap to 80c. 

```suggestion
    """Overlay that displays a polygon on a scene.

    This overlay was created for drawing polygons on Labels layers. It handles
    the following mouse events to update the overlay:
    - Mouse move: Continuously redraw the latest polygon point with the current
    mouse position.
    - Mouse press (left button): Adds the current mouse position as a new
    polygon point.
    - Mouse double click (left button): If there are at least three points in
    the polygon and the double-click position is within completion_radius
    from the first vertex, the polygon will be painted in the image using the
    current label.
    - Mouse press (right button): Removes the most recent polygon point from
    the list.
```